### ISSUE TITLE
Problem initializers

### ISSUE THREAD

**Issue by tomberland** [alokard/SimpleKVO#2]
Hello,

I have the error "observable string cannot be constructed because it has no accessible initializers" on lines `let name = Observable<String>("")` and `let canFly = Observable<Bool>(false)`. Do you know what could be the problem?

Many thanks in advance.
Thomas

**Comment by alokard** [comment]
Hello Thomas,

Can you please provide more details about your case? What version of swift are you using? Are you using cocopods, carthage or manual installation?

Thanks

**Comment by tomberland** [comment]
Hello,

I'm using CocoaPods. I have Xcode Version 7.3.1.
I created a "Bird.swift" file in my iOS project like this :
`import SimpleKVO

class Bird {
    let name = Observable<String>("")
    let canFly = Observable<Bool>(false)
}`

And in my view controller I wrote something like :
`class ViewController: UITableViewController {
    
    var bird = Bird()

    func birdNameChanged(oldValue: String, newValue: String) {
        print("enter birdNameChanged")
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        bird.name.didChange.addHandler(self, handler: #selector(birdNameChanged))
        bird.name = "Pinguin"

        //wait 2 seconds
        let time = dispatch_time(dispatch_time_t(DISPATCH_TIME_NOW), 2 * Int64(NSEC_PER_SEC))
        dispatch_after(time, dispatch_get_main_queue()) {
            bird.name = "Cat"
        }
        ...
    }
    ...
}`

I hope this would help you.

**Comment by alokard** [comment]
Oh, I see where the issue is. As far as we are using `Observable<T>` type for name property in Bird class we can't assign strings to it (`bird.name = "Pinguin"` in your code snippet). There were added special custom operator (`<-`) for setting values for such observable properties (instead of using `set(newValue: T)`).

So for your code snippet you should use next: `bird.name <- "Pinguin"` & `bird.name <- "Cat"`. 
Please try it.

**Comment by tomberland** [comment]
I had the following error : `Use of unresolved operator '<-'`
I assumed there were an error with my install with CocoaPods so I downloaded your project and copied the folder "Source" to my project. That was it, I do not have those errors anymore

But I have now this error :
`Cannot convert value of type 'Selector' to expected argument type '(_) ->(String, String) -> ()'`
 
On the line :
`bird.name.didChange.addHandler(self, handler: #selector(birdNameChanged))`

Thanks for your quick answers :)

**Comment by alokard** [comment]
Yeah, sorry. My bad. There is an issue in read me. In Swift 2.2 syntax #selector notation should be next: `#selector(self.birdNameChanged(_:newValue:)`.

So the line with adding handlers for value changes should look like this:
```swift
bird.name.didChange.addHandler(self, handler: #selector(self.birdNameChanged(_:newValue:))
```

I'll take a look on issue with CocoPods later. Thanks for your feedback.

**Comment by tomberland** [comment]
I still had the error :
`Cannot convert value of type 'Selector' to expected argument type '(_) ->(String, String) -> ()'`

But I found a solution, it's working with this line :

`bird.name.didChange.addHandler(self, handler: { (action) -> (String, String) -> () in self.birdNameChanged(_:newValue:)})`

Thanks for all
